setup.py

```diff
@@ -9,16 +9,18 @@
 
 setup(
     name="advent-of-code-data",
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, !=3.6.*",
```

Comment:
I'd rather not include this restrictive metadata. I just checked with Python 3.5 and Python 3.6 and aocd still works fine, so I see no good reason to explicitly exclude them. I'm not sure the lower bound, but I prefer "innocent until proven guilty" in the `python_requires` constraints.

Reply:
Fair enough. Would you like to keep any of this? Or just remove entirely?

Reply:
We can just remove this line. When I eventually make aocd python3-only, I'll add an appropriate lower bound here.

Reply:
```suggestion
```